src/moveit_python/planning_scene_interface.py

```diff
@@ -96,6 +89,26 @@ def __init__(self, frame, init_from_service=True):
                          PlanningScene,
                          self.sceneCb)
 
+    ## @brief Sent new update to planning scene
+    def sentUpdate(self, collision_object, attached_collision_object, wait=True):
+        ps = PlanningScene()
+        ps.is_diff = True
+        if collision_object:
+            ps.world.collision_objects.append(collision_object)
+
+        if attached_collision_object:
+            ps.robot_state.attached_collision_objects.append(attached_collision_object)
+
+        if wait:
+            resp = self._apply_service.call(ps)
+            if resp.success == True:
+                rospy.loginfo("added planning diff.")
+            else:
+                rospy.logerr("could not add planning diff.")
+            self.waitForSync()
```

Comment:
I would very much prefer to remove this additional `waitForSync` here.
I believe if the user really wants to wait .5 seconds for the next planning scene update, they can call `waitForSync()` after they called all their update methods.

The sync-approach in this package is misleading anyway. [As explained before](https://github.com/ros-planning/moveit/issues/99#issuecomment-337721613), `waitForSync` does not - and cannot - assure *other Subscribers* apart from the `sceneCb` callback have been processed yet. It is not guaranteed that any other subscriber to `move_group/monitored_planning_scene` in the same node or other nodes is already synced.